altair/vegalite/v5/schema/_typing.py

```diff
@@ -38,9 +88,67 @@
     "exponentialb",
 ]
 Align_T: TypeAlias = Literal["left", "center", "right"]
+AllSortString_T: TypeAlias = Literal[
+    "ascending",
+    "descending",
+    "x",
+    "y",
+    "color",
+    "fill",
+    "stroke",
+    "strokeWidth",
+    "size",
+    "shape",
+    "fillOpacity",
+    "strokeOpacity",
+    "opacity",
+    "text",
+    "-x",
+    "-y",
+    "-color",
+    "-fill",
+    "-stroke",
+    "-strokeWidth",
+    "-size",
+    "-shape",
+    "-fillOpacity",
+    "-strokeOpacity",
+    "-opacity",
+    "-text",
+]
 AutosizeType_T: TypeAlias = Literal["pad", "none", "fit", "fit-x", "fit-y"]
 AxisOrient_T: TypeAlias = Literal["top", "bottom", "left", "right"]
 Baseline_T: TypeAlias = Literal["top", "middle", "bottom"]
+BinnedTimeUnit_T: TypeAlias = Literal[
```

Comment:
This was the alias that I was aiming for when I opened this and below is an example of where it would appear.

https://github.com/vega/altair/blob/8b9f6978c6f1b27fd50a4e3a509aecb9b4a94236/altair/vegalite/v5/schema/core.py#L4525-L4563

Everything else was a bonus